### PR TITLE
Fix no-schedule issue

### PR DESCRIPTION
**Pull request by christian-schlichtherle**
The existing manifest doesn't work due to the proposed node taint. This fixes it.

### CODE REVIEW

**Comment by christian-schlichtherle** [comment]
> Thanks for your pull request. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please follow instructions at https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.**
> 
> It may take a couple minutes for the CLA signature to be fully registered; after that, please reply here with a new comment and we'll verify. Thanks.
> 
> * If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address.  Check your existing CLA data and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * If you signed the CLA as a corporation, please sign in with your organization's credentials at https://identity.linuxfoundation.org/projects/cncf to be authorized.
> * If you have done the above and are still having issues with the CLA being reported as unsigned, please log a ticket with the Linux Foundation Helpdesk: https://support.linuxfoundation.org/
> * Should you encounter any issues with the Linux Foundation Helpdesk, send a message to the backup e-mail support address at: [login-issues@jira.linuxfoundation.org](mailto:login-issues@jira.linuxfoundation.org)

done

File: docs/gpu.md [hunk 1]
```diff
@@ -41,35 +41,52 @@ In order to match the kOps environment, create a `values.yaml` file with the fol
 operator:
   nodeSelector:
     kops.k8s.io/instancegroup: gpu-nodes
+  tolerations:
+    - key: nvidia.com/gpu
+      operator: Exists
```
Comment (hakman) [inline docs/gpu.md:46]: Please don't indent the sequence values, to keep things consistent. Same in other places.
```suggestion
  - key: nvidia.com/gpu
    operator: Exists
```

**Review by hakman — CHANGES_REQUESTED** [review]

File: docs/gpu.md (hunk 1 above)
Comment (christian-schlichtherle) [reply docs/gpu.md]: done

Comment (hakman) [reply docs/gpu.md]: Thanks. Please also squash the commits.

Comment (christian-schlichtherle) [reply docs/gpu.md]: I'm fed up with the bureaucratic overhead in this repository for a simple change like this. Take it or leave it - no offense.

Comment (hakman) [reply docs/gpu.md]: No offense taken, sorry you feel this way. Thank you for your contribution.

**Pull request merged**